### ISSUE TITLE
Progress Bar when video is buffering.

### ISSUE THREAD

**Issue by vinit-kanani** [ram-on/SkyTube#127]
It would be nice if there is a progress bar when the video starts to buffer in the middle.

**Comment by hamishmb** [comment]
Agreed, it would be good to have a throbber or progress bar when buffering, at the start or later.
